charts/newrelic-infrastructure-beta/templates/configmap.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if .Values.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}
+data:
+    newrelic-infra.yml: |
+{{ toYaml .Values.config | indent 6 }}
+{{ end }}
+{{ if .Values.integrations_config }}
```

Comment:
I would split this CM in two: `integrations-configmap` and `agent-configmap`. Moreover `integrations-configmap` could me moved into the `kubelet` folder.